virttest/qemu_devices/qcontainer.py

```diff
@@ -1082,8 +1087,14 @@ def pflash_handler(firmware_name, cmd):
                 else:
                     img_path, img_name = os.path.split(img_info.get("filename"))
 
-                    pflash_vars_name = "%s_%s_VARS.fd" % (self.vmname, img_name)
-                    pflash_vars_path = os.path.join(img_path, pflash_vars_name)
+                    pflash_vars_name = "%s_%s_%s_%s_VARS.fd" % \
+                       (self.vmname, img_name,
+                        params['image_backend'], img_info.get("format"))
```

Comment:
Shall we need to rename the `pflash_vars_name` for the regular image?
I mean weather to add the external image info of `image_backend` and `format`.
Hi, @PaulYuuu 
Would you like to give some comments?
Thanks.

Reply:
> I think `img_name` already contains the image format, and for `image_backend`, in localFS, they should always `filesystem`? I am not sure, it's okay for me to add this key or not

Okay, thanks.